src/patterns/task-list-pages/index.md.njk

```diff
@@ -54,9 +56,11 @@ Where possible, task names should:
 
 ### Show the status of the tasks
 
-Include a summary above the task list to say how many tasks or sections have been completed.
+Include a summary above the task list to say how many tasks have been completed.
+
+You might find a better word to describe what a ‘task’ is for your users. If your service is an application form, each task might be a ‘section’ of questions to complete.
 
-![A screenshot showing a task list summary that says 'Application incomplete. You have completed 3 of 8 sections'](task-list-summary.png)
+![A screenshot showing a task list summary that says 'Application incomplete. You've completed 3 of 8 sections'](task-list-summary.png)
```

Comment:
The screenshot (task-list-summary.png) needs an update to use the contraction ('you've' instead of 'you have'). Is this one for you @frankieroberto?

Reply:
@calvin-lau-sig7 this pre-exists so I don't have the original. Could we revert the alt text and tackle this as a separate PR? Possibly the screenshot could be converted to HTML, or we could just remove it and the sentence above, as it doesn't seem to be widely followed so far?

Reply:
Ahh understood. If we can work to change it, I'd say we can leave the screenshot in for now.
I just realised that the larger screenshot earlier on (task-list-whole.png) also uses the same language.

I'll approve now for merge.

Reply:
@calvin-lau-sig7 thanks!